beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -290,10 +297,10 @@ function getCrosslinkCommitteesAtSlot(state: BeaconState, slot: SlotNumber, regi
     }
   }
 
-  const shuffling = getShuffling(seed, state.validatorRegistry, shufflingEpoch);
-  const offset = slot % EPOCH_LENGTH;
+  const shuffling: ValidatorIndex[][] = getShuffling(seed, state.validatorRegistry, shufflingEpoch);
+  const offset: int = slot.umod(new BN(EPOCH_LENGTH)).toNumber();
   const committeesPerSlot = Math.floor(committeesPerEpoch / EPOCH_LENGTH);
```

Comment:
Swap `Math.floor(committeesPerEpoch / EPOCH_LENGTH)`

for `committeesPerEpoch.divn(EPOCH_LENGTH)`